### ISSUE TITLE
New UI and searched results

### ISSUE THREAD

**Issue by Rudresh-pandey** [aditya-singh9/kekfinder#46]
### **Current View :** 
After adding the above feature ---
![image](https://user-images.githubusercontent.com/96531798/193435737-704a4751-356c-4376-bdc0-a49a25a94aa2.png)

_**This has also some issue like overlaping of emoji's and multiple select features. which had already been noticed **_
.
.
.
### **Light Theme :**
![image](https://user-images.githubusercontent.com/96531798/193435871-1b5b3f11-faeb-4bba-8842-83a169c341cd.png)

### **Dark Theme :**
![image](https://user-images.githubusercontent.com/96531798/193435929-6e1a7cd9-0e56-4017-a596-544e454920b8.png)

###  **And another suggestion on what to show when nothing is searched -- Rather than creating favourite section or showing all the emoji's , We can show all the previously copied emoji's to the user so that we don't have to bother about the number of emoji's to be displayed.**

_I am giving the link to the Figma design , feel free to make changes and present your idea_
https://www.figma.com/file/MLuUOQ38c7UC3Qh7dT0ekc/kekFinder?node-id=5%3A31
 :)

**Comment by aditya-singh9** [comment]
The part of, what to di when nothing is searched is being covered by another contributor so you dont have tk worry about that
You can start working on the ui fixes

**Comment by Rudresh-pandey** [comment]
working on UI :)

**Comment by Substancia** [comment]
Hey @Rudresh-pandey is this a critical work, and if I proceed with https://github.com/aditya-singh9/kekfinder/issues/41, will it be breaking changes to your work?

**Comment by Rudresh-pandey** [comment]
Yes let me do the changes first .

**Comment by Substancia** [comment]
Sure. Let me know once you're done, and I'll proceed.

**Comment by Rudresh-pandey** [comment]
bhai , hum Ui improve kar diye h lekin ab ek do orr merge ho gaya ab kaise PR bheje

**Comment by aditya-singh9** [comment]
Iguess you have to sync your fork

Or i think make a new branch and push the changes to that branch and then create a pr

I do no know exactly what we have to do.

**Comment by aditya-singh9** [comment]
Did you fix it?

I wont merge anything from now till i merge your pr

**Comment by Rudresh-pandey** [comment]
bro , ye changes nhi ho paaega kuki jab starting mai user enter karga tab emoji's 0 rahega or tab UI thik nhi laega

![image](https://user-images.githubusercontent.com/96531798/193507068-81b73245-249c-43ba-895a-dcd6a1dbb7b5.png)

isse leye close kar diye isko :) sorry message thoda late se kar rahe hai

**Comment by aditya-singh9** [comment]
It is being taken care by @Substancia at https://github.com/aditya-singh9/kekfinder/issues/60
so don't worry about it
make a pr

**Comment by Rudresh-pandey** [comment]
bro wo tho tab show hoga jab user pahle se emoji's ko pahle se copy kiya ho

**Comment by aditya-singh9** [comment]
Now see the website.
Now it is showing

**Comment by Rudresh-pandey** [comment]
On it sir .. 
Great work everyone

**Comment by aditya-singh9** [comment]
Okay i wont merge anycode in this repo tmrw until your pr is made and merged

**Comment by Substancia** [comment]
There shouldn't be any merge conflicts if we are periodically syncing our fork with the upstream (or original repo). Technically blocking other PRs isn't ideal and goes against the concept of Git, but since we don't have many critical works running in parallel right now, I guess this is doable.

My point is, please periodically sync your fork with the original repo to avoid merge conflicts, so other devs won't have their work blocked. Cheerio!

**Comment by aditya-singh9** [comment]
@Rudresh-pandey now please sync the repo and then start working on it

**Comment by Rudresh-pandey** [comment]
phone view wala code kidhr hai? desktop wala pe changes kiye tho mobile view wala change ho ja raha h uska code kaha hai?

**Comment by aditya-singh9** [comment]
What do you mean by the phone code?

I do not know where it is because i havent seen the project for 10month

It must be in some css file

**Comment by Rudresh-pandey** [comment]
ok

**Comment by Rudresh-pandey** [comment]
**@aditya-singh9  bro  laptop ka screen mai thik dikh raha h ,**

https://user-images.githubusercontent.com/96531798/193795935-c9d59775-34a9-4589-966d-e2b081985c5d.mp4

**Lekin responsive nhi ho raha** 

https://user-images.githubusercontent.com/96531798/193796047-48bf89e3-9848-45a3-b91b-6ad9b489ebd9.mp4

_Responsive hone ka code node modules mai hai usse kasie karte hai mujhe nhi pata_  i guess iss issue ko close karke dusra method dhundna hoga wo sticky wala issue ko solve karne ke leye #64 ya koi orr responsive code pe kaam kar sakta hai agar tho . 
@Substancia @ArunBohra12  any suggestions ?

**Comment by Substancia** [comment]
One way to tackle this is creating responsiveness using CSS media queries. You can define a screen width which differentiates different devices, and style the container accordingly for each screen.

As for mobile view, maybe we can stick with old design of having emojis below? In which case the [sticky section issue](https://github.com/aditya-singh9/kekfinder/issues/64) will be needed to be addressed.


Also there has been a [PR merge](https://github.com/aditya-singh9/kekfinder/issues/61#issuecomment-1266716423) (purely functional), which will not conflict your changes. So please, before you create your PR, sync your fork once and pull changes to your branch once.

**Comment by aditya-singh9** [comment]
How can the code of responsiveness be be in node modules

The code must be components folder. separate components have their separate css files, therefore the css of separate components will be in thier respective folders

**Comment by aditya-singh9** [comment]
> One way to tackle this is creating responsiveness using CSS media queries. You can define a screen width which differentiates different devices, and style the container accordingly for each screen.
> 
> As for mobile view, maybe we can stick with old design of having emojis below? In which case the [sticky section issue](https://github.com/aditya-singh9/kekfinder/issues/64) will be needed to be addressed.

Yes iguess in the mobile view, we should stick to the older design

**Comment by ArunBohra12** [comment]
Yes IMO, media queries will be the best.

**Comment by ArunBohra12** [comment]
> **Lekin responsive nhi ho raha**
> _`Responsive hone ka code node modules mai hai usse kasie karte hai mujhe nhi pata`_ i guess iss issue ko close karke dusra method dhundna hoga wo sticky wala issue ko solve karne ke leye #64 ya koi orr responsive code pe kaam kar sakta hai agar tho . @Substancia @ArunBohra12 any suggestions ?

Where do you see the code in node modules?

**Comment by Rudresh-pandey** [comment]
![image](https://user-images.githubusercontent.com/96531798/193839043-0ba93865-7d97-42db-b128-f6a6bbe7f53a.png)

and ToastContainer using toastContainer.scss (**node modules -> react-toastify -> scss -> animations** )

**Comment by Substancia** [comment]
> ![image](https://user-images.githubusercontent.com/96531798/193839043-0ba93865-7d97-42db-b128-f6a6bbe7f53a.png)
> 
> and ToastContainer using toastContainer.scss (**node modules -> react-toastify -> scss -> animations** )

These are certain default styles coming from certain packages, and are overridden by the css files that we define, don't have to worry about them IF you're writing the styles that we need (just pretend that the default media queries don't exist and start writing).

TL;DR: write your own media queries for responsiveness and these styles from node modules will disappear.

**Comment by Rudresh-pandey** [comment]
ok thanks :)

**Comment by Rudresh-pandey** [comment]
### **Have a look :-**

https://user-images.githubusercontent.com/96531798/193889062-9020a03c-d531-4388-9e46-b43258e78055.mp4


### **This will solve some issues :**

`1 . The text on the search bar is not visible completely`

**Old View:**
![image](https://user-images.githubusercontent.com/96531798/193889373-b5e1cea7-ed0f-4247-9c10-8667c5434740.png)

**New View:**
![image](https://user-images.githubusercontent.com/96531798/193889500-cf0e90ff-5e9f-4b22-a3c5-177989220ca2.png)

`2 . The issue of the sticky search bar and multiple select feature(msf)  - in desktop view .`
_We dont't have to worry about the search bar and msf in mobile view because we have a feature ScrollToTop (see the video)_ 

`3 . Number of emoji's in a row :` _more to see less to scroll_

**Old View :**
![image](https://user-images.githubusercontent.com/96531798/193890406-8b5e3aa2-dec5-482c-b452-50ad09d5e243.png)


**New View :**
![image](https://user-images.githubusercontent.com/96531798/193890512-68cfa697-a618-46ac-b64a-ed95fef78559.png)


thanks  - 
@aditya-singh9 
@Substancia 
@ArunBohra12  , for helping me out

**Comment by Substancia** [comment]
Good job @Rudresh-pandey.

https://github.com/aditya-singh9/kekfinder/issues/46#:~:text=2%20.%20The%20issue%20of%20the%20sticky%20search%20bar%20and%20multiple%20select%20feature(msf)%20%2D%20in%20desktop%20view%20.%0AWe%20dont%27t%20have%20to%20worry%20about%20the%20search%20bar%20and%20msf%20in%20mobile%20view%20because%20we%20have%20a%20feature%20ScrollToTop%20(see%20the%20video)

Tbh I'd prefer as a user to have the selected emojis section visible at all times, including when we scroll down, for the ease of keeping track what all emojis I've selected so far. This way, I can also keep track of misclicks and wrong emojis and immediately delete the wrong emojis from the selection. In which case, the [sticky situation](https://github.com/aditya-singh9/kekfinder/issues/64) has to be taken care of as well.

Otherwise, the UI looks good. What does @aditya-singh9 think?

**Comment by Rudresh-pandey** [comment]
Yes we have to keep track of emoji's . Working on it :)

**Comment by aditya-singh9** [comment]
I really appreciate his work and i really like the ui too.

And for the stickness, i am not sure, what should we do.
Like should we keep it or remove it

**Comment by Rudresh-pandey** [comment]
Can i merge the current code then we should have a discussion whether to keep the MSF or not . 
:)

**Comment by aditya-singh9** [comment]
Yes, please merge the current code.

And about the stickness it will be taken care in new pr and we can have discussion about it in https://github.com/aditya-singh9/kekfinder/issues/64
